cmd/medius/common/registry.go

```diff
@@ -0,0 +1,32 @@
+package common
+
+import (
+	"kubevirt.io/containerdisks/artifacts/centos"
+	"kubevirt.io/containerdisks/artifacts/fedora"
+	"kubevirt.io/containerdisks/artifacts/rhcos"
+	"kubevirt.io/containerdisks/pkg/api"
+)
+
+type Entry struct {
+	Artifact   api.Artifact
+	UseForDocs bool
+}
+
+var Registry = []Entry{
+	{
+		Artifact:   fedora.New("35"),
+		UseForDocs: true,
+	},
+	{
+		Artifact:   rhcos.New("4.9"),
+		UseForDocs: true,
+	},
+	{
+		Artifact:   centos.New("8.4"),
+		UseForDocs: true,
+	},
+	{
+		Artifact:   centos.New("7-2009"),
+		UseForDocs: false,
+	},
```

Comment:
Does this order mean that CentOS 7 doc overwrites CentOS 8 doc?

Reply:
CentOS8 docs are with this setting the mandatory source for the docs, since they share a repo.

Reply:
I see, this is why it is called `UseForDocs`. sorry for the noise.